test-requirements.txt

```diff
@@ -1 +1,2 @@
 pytest==8.3.4
+great_expectations==1.3
```

Comment:
this shouldn't be required, GX is already specified in the project deps in `setup.cfg`

Reply:
Yeah, not sure why I thought things were failing without that 🤷 . Removed!